### PR TITLE
Added pronunciation part on main-page

### LINKED ISSUE #153: openHAB pronunciation
There should be a pronunciation information on the starting page of docs.

* https://en.wikipedia.org/wiki/International_Phonetic_Alphabet
* http://www.i2speak.com
* http://www.ipachart.com

Maybe an easier solution would be nice ;)

### PR DESCRIPTION
**Pull request by BClark09**
Closes #153 

Signed-off-by: Ben Clark <ben@benjyc.uk>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 <div class="container target">
 	<h1>Welcome to the openHAB 2 Documentation!</h1>
+	<p>The <strong>open H</strong>ome <strong>A</strong>utomation <strong>B</strong>us (openHAB, <em>pronounced ˈəʊpənˈhæb</em>) is an open source, technology agnostic home automation platform which runs as the center of your smarthome!</p>
```
Comment (ThomDietrich) [inline index.html]: http://docs.openhab.org/images/logo.png
I think it would be more in line to write "smart home"

File: index.html [hunk 2]
```diff
@@ -7,6 +7,7 @@
...
 	<p>This documentation contains different sections. If you are new to openHAB 2, you might want to start with one of our <a href="tutorials">Tutorials</a> that guides you through different steps by an example. If you are looking for some specific information on how to use openHAB, the <a href="overview.html">User Manual</a> is the right place to go and if you plan to get started with your own developments, please refer to our <a href="developers">Developer Guide</a>.</p>
```
Comment (ThomDietrich) [inline index.html]: While you are at it. The first sentence seems like an awkward mixture of plural and singular

**Comment by BClark09** [comment]
Done! :)

**Comment by ThomDietrich** [comment]
Wonderful :smiley:

**Pull request merged**